src/jit/function-builder.cc

```diff
@@ -693,6 +693,76 @@ bool FunctionBuilder::Emit(TR::BytecodeBuilder* b,
       break;
     }
 
+    case Opcode::I32Load8S:
+      Push(b,
+           "i32",
+      b->  ConvertTo(Int32,
+      b->            LoadAt(typeDictionary()->PointerTo(Int8), EmitMemoryPreAccess<int8_t>(b, &pc))));
+      break;
+
+    case Opcode::I32Load8U:
+      Push(b,
+           "i32",
+      b->  UnsignedConvertTo(Int32,
+      b->            LoadAt(typeDictionary()->PointerTo(Int8), EmitMemoryPreAccess<int8_t>(b, &pc))));
```

Comment:
Just a small nitpick, but the indentation is wrong on all your `UnsignedConvertTo`s for unsigned extending loads